tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -889,11 +938,12 @@ def _print_sts_counters(spinner, cloud_logger, counters, is_job_done):
                     bytes_copied_to_sink, objects_copied_to_sink)
             else:
                 new_text = (
-                    'Error! STS job copied {} of {} bytes in {} of {} objects and deleted'
```

Comment:
Why is this commented out and marked as success when byte_status and object_status are not True.  If the idea is to handle retain_source_bucket config remove the bytes_deleted_from_source and objects_deleted_from_source counters while calculating byte_status and object_status as deleted info is not highly important.